mcux/drivers/imx/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ zephyr_library_sources_ifdef(CONFIG_UART_MCUX_LPUART	fsl_lpuart.c)
 zephyr_library_sources_ifdef(CONFIG_VIDEO_MCUX_CSI	fsl_csi.c)
 zephyr_library_sources_ifdef(CONFIG_WDT_MCUX_IMX_WDOG    fsl_wdog.c)
 zephyr_library_sources_ifdef(CONFIG_CAN_MCUX_FLEXCAN	fsl_flexcan.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_MCUX_12B1MSPS_SAR	fsl_adc.c)
```

Comment:
Please move this up to preserve alpha order by CONFIG

Reply:
updated in last commit